include/bluetooth/audio/audio.h

```diff
@@ -1516,6 +1516,45 @@ int bt_audio_unicast_group_create(struct bt_audio_stream *streams,
 				  uint8_t num_stream,
 				  struct bt_audio_unicast_group **unicast_group);
 
+/** @brief Add streams to a unicast group as a unicast client
+ *
+ *  This function can be used to add additional streams to a
+ *  bt_audio_unicast_group.
+ *
+ *  This can be called at any time before when none of the streams in the
+ *  group has not been started (see bt_audio_stream_ops.started()).
```

Comment:
Typo:
```suggestion
 *  This can be called at any time before any of the streams in the
 *  group has been started (see bt_audio_stream_ops.started()).
```